docs/examples/JSONExample/src/JSONExample.ls

```diff
@@ -90,11 +123,51 @@ package
             sprite.x = 240;
             sprite.y = 120;
             stage.addChild(sprite);
+            
+            
+            var json = new JSON();
+            
+            json.initObject();
+            
+            
+            // Circular reference tests
```

Comment:
Oh OK. So we should reinstate it when on a newer version of Loom? Is there a way we can put an if around it to automatically enable it on "good" versions of Loom?